lib/v2/cuc/yz.js

```diff
@@ -0,0 +1,38 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const url = require('url');
+const iconv = require('iconv-lite');
+
+const host = 'http://yz.cuc.edu.cn/';
+
+/* 研究生招生网通知公告*/
+
+module.exports = async (ctx) => {
+    const response = await got({
+        method: 'get',
+        url: host,
+        responseType: 'buffer',
+    });
+
+    const responseHtml = iconv.decode(response.data, 'gbk');
+    const $ = cheerio.load(responseHtml);
+
+    const notice = $('#notice_area').children('.notice_block_top');
+    const content = notice.children('.notice_content1').children('p');
+    const items = content
+        .map((_, elem) => {
+            const a = $('a', elem);
+            return {
+                link: url.resolve(host, a.attr('href')),
```

Comment:
URL.resolve is deprecated. please use wga

Reply:
I haven`t use wga,could you please to show me the doc`s url?

Reply:
"URL.resolve is deprecated. please use whatwg API instead"
https://nodejs.org/api/url.html